tests/functional/metadata/MixedVersionFormat.js

```diff
@@ -97,7 +97,9 @@ describe('Mongo backend mixed bucket format versions', () => {
             });
         });
 
-        after(done => mongoClient.close(true).then(() => done()).catch(() => done()));
+        after(() => {
+            mongoClient.close(true);
+        });
 
```

Comment:
mongodClient.close() is async (ie returns a Promise), so need to make the function passed to after async as well: too ensure it waits until mongo client is closed.

```suggestion
        after(() => mongoClient.close(true));
```

or

```suggestion
        after(async () => {
            await mongoClient.close(true);
        });
```